### ISSUE TITLE
Any planned support for CMYK?

### ISSUE THREAD

**Issue by jackparmer** [casesandberg/react-color#215]
We have a few customers at Plotly that prefer to work in CMYK. Any recommendations for how we could support this, since we use this project (and love it!)?

**Comment by casesandberg** [comment]
Would you like there to be a specific CMYK color picker? Or you just want the output in the code to also return CMYK transformations for whatever color there is?

**Comment by jackparmer** [comment]
> Would you like there to be a specific CMYK color picker? Or you just want the output in the code to also return CMYK transformations for whatever color there is?

Thanks for responding! The customer is looking for the former. The developers at Plotly would need the latter as well. So, both 😄

**Comment by casesandberg** [comment]
Awesome! I can do both, which one of the color pickers would you like the text boxes changed to CMYK? I can make one just like that that also returns CMYK!

**Comment by jackparmer** [comment]
![image](https://cloud.githubusercontent.com/assets/1865834/18182049/3a203a8e-705c-11e6-8f0b-f674cd8cb87f.png)

This one! We'd love to have the option to have CMYK input here. Thanks for considering this, we understand this likely won't be a super popular feature 😑 

This company has all their style guides in print format in CMYK, so this would be a huge convenience for them. 

Thanks again for considering!

**Comment by casesandberg** [comment]
Perfect! I will start on this later!

**Comment by benkeen** [comment]
+1!

**Comment by mclavacca** [comment]
+1

**Comment by TobiasTimm** [comment]
Any news on the cmyk support?

**Comment by iamraffe** [comment]
Hate to pretty much say the same than the guy before me, but any news on CMYK support?

**Comment by casesandberg** [comment]
Looks like there is [a pr in tinycolor](https://github.com/bgrins/TinyColor/pull/160/files) that needs to be merged first, but after that it would be trivial to make work!

**Comment by iamraffe** [comment]
Good to hear! Thanks for the update.

**Comment by crertel-packlane** [comment]
Any updates? I poked the PR mentioned upthread.

**Comment by casesandberg** [comment]
Working on v3.0.0 right now, going to track this with that milestone to make sure this is supported!

**Comment by semiadam** [comment]
+1

**Comment by semiadam** [comment]
I just wanted to mention that, since the result of CMYK palette needs to be displayed on the screen as well, there is always an RGB equivalent needed, so the tool needs to simultaneously return the CMYK and its equivalent RGB. The mapping from RGB to CMYK is not 1-to-1 and there are several algorithms for it which gain different accuracy. The most accurate one, as far as I know, is the one Adobe products use. I'm not sure if it could be licensed but at least it can be used for quality control.

**Comment by redlive** [comment]
Any news on the cmyk support?

**Comment by 0ro** [comment]
+1

**Comment by shahzaib-tintash** [comment]
+1

**Comment by simzar** [comment]
+1

**Comment by gherkins** [comment]
@casesandberg how about using https://www.npmjs.com/package/color-convert?

**Comment by joeyparis** [comment]
We would also love to have a CMYK pallet option.

**Comment by exeyy** [comment]
give us the cmyk!

**Comment by khajaamin** [comment]
Yes, this is reason I have to  fallback last time and even now I am not able to see  something happening. 

Is it imposible ?   can somebody guide I can develop with your guidance

**Comment by mgacrama211176** [comment]
+1 No update for the CMYK as of the moment

**Comment by Vesli** [comment]
Any update?
